docs/maintainers/adr/0020-reduce-support-for-internet-explorer.md

```diff
@@ -0,0 +1,77 @@
+# Reduce support for Internet Explorer
+
+* Status: accepted
+* Date: 2023-12-19
+
+Technical Story: PLATUI-2686
+
+## Context and Problem Statement
+
+[govuk-frontend v5.0.0](https://github.com/alphagov/govuk-frontend/releases/tag/v5.0.0)
+drops support for Internet Explorer (IE) versions 8 through 10 by preventing loading of its Javascript (JS)
+and removing various IE-specific polyfills.  IE11 is still supported via graceful degradation to a non-JS experience.
+
+We propose to follow the same approach in our frontend libraries, preventing loading of our own JS and
```

Comment:
Can we define what we mean by `our frontend libraries`, in case future maintainers do not find this clear (`hmrc-frontend` and `play-frontend`hmrc`).

Reply:
Good shout - I've made this explicit where they're first mentioned